src/patternfly/components/Slider/slider.scss

```diff
@@ -0,0 +1,204 @@
+.pf-c-slider {
+  // rail
+  --pf-c-slider__rail--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-slider__rail--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-slider__rail-track--Height: #{pf-size-prem(4px)};
+  --pf-c-slider__rail-track--before--base--BackgroundColor: var(--pf-global--BorderColor--100);
+  --pf-c-slider__rail-track--before--fill--BackgroundColor: var(--pf-global--active-color--300);
+  --pf-c-slider__rail-track--before--BorderRadius: var(--pf-global--BorderRadius--lg);
+  --pf-c-slider__rail-track--before--fill--BackgroundColor--gradient-stop: var(--pf-c-slider--value);
+
+  // steps
+  --pf-c-slider__steps--FontSize: var(--pf-global--FontSize--sm);
+  --pf-c-slider__steps--Height: var(--pf-c-slider__steps--FontSize);
+
+  // step tick
+  --pf-c-slider__step-tick--Top: var(--pf-global--spacer--md);
+  --pf-c-slider__step-tick--Width: #{pf-size-prem(4px)};
+  --pf-c-slider__step-tick--Height: #{pf-size-prem(4px)};
```

Comment:
should the value for height be the variable for width since they're proportional

Reply:
It could, though it creates the potential breaking change where, if we make the track 10px tall and we want the tick to be 4x10, if we change `--pf-c-slider__step-tick--Height: var(--pf-c-slider__step-tick--Width);` to `--pf-c-slider__step-tick--Height: 10px;`, then since `--pf-c-slider__step-tick--Width` used to control `--pf-c-slider__step-tick--Height`, and it no longer does, that could be a breaking change. Similar to how we have hover/focus/active, and focus/active aren't defined by the hover var, even though they're all the same value.

Though in this case, I don't think it's likely, but it doesn't cost much to keep them independent.

Reply:
ok makes sense lets keep them